common/test_gcloud.py

```diff
@@ -170,9 +170,9 @@ def test_delete_instances_greater_than_batch_size(mocked_execute):
   than batch size."""
     instances = ['instance-%d' % i for i in range(103)]
     mocked_execute.return_value = new_process.ProcessResult(0, '', False)
-    # -q is needed otherwise gcloud will prompt "Y/N?".
```

Comment:
Why delete this?

Reply:
It is there in source code in gcloud.delete_instances, not needed in test again.